libraries/AP_ExternalAHRS/AP_ExternalAHRS_AnelloEVK.h

```diff
@@ -148,6 +153,7 @@ class AP_ExternalAHRS_AnelloEVK: public AP_ExternalAHRS_backend
     double extract_double(const uint8_t* data, uint8_t offset) const;
     float extract_float(const uint8_t* data, uint8_t offset) const;
     Quaternion populate_quaternion(const uint8_t* data, uint8_t offset) const;
+    uint8_t data_packet_length(const AnelloEVK_Packet & packet) const;
```

Comment:
Is this a bitwise and? Is it supposed to be passed by reference? See 571 of the cpp file.